app/src/main/java/org/jboss/hal/client/configuration/subsystem/elytron/OtherSettingsView.java

```diff
@@ -66,6 +75,9 @@
 
     private OtherSettingsPresenter presenter;
 
+    private static final Constants CONSTANTS = GWT.create(Constants.class);
```

Comment:
We should use static constants for `Constants` and `Messages` only if there's no DI available. Please inject `org.jboss.hal.resources.Resources` instead, which gives you access to all kind of resources (i18n, preview texts, images, ...)